### PR TITLE
Add FTL arrival visuals

### PR DESCRIPTION
**Pull request by metalgearsloth**
Requires https://github.com/space-wizards/RobustToolbox/pull/5265

https://github.com/space-wizards/space-station-14/assets/31366439/e76ce146-e647-4c00-ab50-1fdc6947c196

:cl:
- add: Add effects for when shuttles are arriving.

### CODE REVIEW

**Comment by MACMAN2003** [comment]
finally, no more jumpscares!

File: Content.Shared/Shuttles/Components/FTLComponent.cs [hunk 1]
```diff
@@ -29,13 +29,19 @@ public sealed partial class FTLComponent : Component
...
+    public EntProtoId? VisualizerProto = "FtlVisualizerEntity";
+
+    [DataField, AutoNetworkedField]
+    public EntityUid? VisualizerEntity;
+
     /// <summary>
     /// Coordinates to arrive it: May be relative to another grid (for docking) or map coordinates.
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite), DataField]
+    [ViewVariables(VVAccess.ReadWrite), DataField, AutoNetworkedField]
```
Comment (Tayrtahn) [inline Content.Shared/Shuttles/Components/FTLComponent.cs:41]:
```suggestion
    [DataField, AutoNetworkedField]
```

**Comment by ps3moira** [comment]
Does this work with EVAC and any other ship FTLing?

**Comment by MerrytheManokit** [comment]
I feel like if this is added, it shouldn’t affect the nukie ship and be justified as some sort of stealth drive. Just in the off chance someones sees it.

**Comment by metalgearsloth** [comment]
> I feel like if this is added, it shouldn’t affect the nukie ship and be justified as some sort of stealth drive. Just in the off chance someones sees it.

I imagine seeing the nukie ship 5 seconds later to be a bigger issue.

> Does this work with EVAC and any other ship FTLing?

Anything that uses FTL code.

**Comment by ps3moira** [comment]
> > I feel like if this is added, it shouldn’t affect the nukie ship and be justified as some sort of stealth drive. Just in the off chance someones sees it.
> 
> I imagine seeing the nukie ship 5 seconds later to be a bigger issue.
> 
> > Does this work with EVAC and any other ship FTLing?
> 
> Anything that uses FTL code.

awesome

**Pull request merged**